cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/connector/ConnectorSpec.java

```diff
@@ -31,10 +31,22 @@ public class ConnectorSpec {
   // schema is null when the connector is unable to retrieve it from the resource
   private final Schema schema;
   private final Set<PluginSpec> relatedPlugins;
+  private final Set<String> supportedSampleTypes;
```

Comment:
Why is it String? I would expect it to be an Enum or at least otherwise proper values should be dictated by the platform

Reply:
Funny you mention this -- we're actually in the middle of moving it over to an Enum, but we just wanted to get the PR out for review sooner rather than later. It'll be changed soon!

Reply:
Just merged into this branch